docs/integrations/microsoft-azure/azure-key-vault.md

```diff
@@ -33,24 +33,98 @@ When you configure the event hubs source or HTTP source, plan your source catego
 
 ### Configure metrics collection
 
-In this section, you will configure a pipeline for shipping metrics from Azure Monitor to an Event Hub, on to an Azure Function, and finally to an HTTP Source on a hosted collector in Sumo Logic.
+:::note
+Sumo Logic Metrics source is currently in Beta, to participate, contact your Sumo Logic account executive.
+:::
+
+In the Sumo Logic Azure Metrics source configuration,
+
+- Tag the location field in the source with correct Azure resource location value. <br/><img src={useBaseUrl('img/integrations/microsoft-azure/Azure-Storage-Tag-Location.png')} alt="Azure Key Vaults Location" style={{border: '1px solid gray'}} width="400" />
```

Comment:
this step is not needed since location is present already in metrics we can verify this in our source.
Also remove it from container app as well if it's present there.